src/rdb.c

```diff
@@ -2762,7 +2763,6 @@ int rdbFunctionLoad(rio *rdb, int ver, functionsCtx* functions_ctx) {
     if (engine_name) sdsfree(engine_name);
     if (desc) sdsfree(desc);
     if (blob) sdsfree(blob);
-    if (err)  sdsfree(err);
     return res;
```

Comment:
maybe make sure to nullify `err` on success?
maybe make `err` an optional argument?

Reply:
OK, will make it optional.